test/elixir_internal_certificate_web/controllers/user_session_controller_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule ElixirInternalCertificateWeb.UserSessionControllerTest do
+  use ElixirInternalCertificateWeb.ConnCase, async: true
+
+  import ElixirInternalCertificate.AccountsFixtures
```

Comment:
Hi @nvminhtue first comment, we are using ExMachina to generate the test data in most of our projects, I think you can spend time checking it https://hex.pm/packages/ex_machina

Reply:
@andyduong1920 thank you, I'm reaching out.